pkg/util/mount/mount_linux.go

```diff
@@ -1003,10 +1003,11 @@ func (mounter *Mounter) SafeMakeDir(subdir string, base string, perm os.FileMode
 }
 
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
-	if _, err := os.Stat(pathname); os.IsNotExist(err) {
+	pathExists, pathErr := PathExists(pathname)
+	if !pathExists || IsCorruptedMnt(pathErr) {
```

Comment:
maybe add a log for IsCorruptedMnt for debugging?

Reply:
resolved